src/main/java/thePackmaster/cards/fueledpack/HotAsh.java

```diff
@@ -0,0 +1,39 @@
+package thePackmaster.cards.fueledpack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.LoseHPAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.util.Wiz.adp;
+import static thePackmaster.util.Wiz.atb;
+
+public class HotAsh extends AbstractFueledCard {
+    public final static String ID = makeID(HotAsh.class.getSimpleName());
+    private static final CardType TYPE = CardType.ATTACK;
+    private static final CardRarity RARITY = CardRarity.SPECIAL;
+    private static final CardTarget TARGET = CardTarget.ENEMY;
+    private static final int COST = 1;
+
+    private static final int DAMAGE = 5;
+    private static final int UPGRADE_DAMAGE = 3;
+    private static final int MAGIC = 2;
+
+    public HotAsh() {
+        super(ID, COST, TYPE, RARITY, TARGET, CardColor.COLORLESS);
+        baseDamage = DAMAGE;
+        baseMagicNumber = magicNumber = MAGIC;
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        dmg(m, AbstractGameAction.AttackEffect.FIRE);
+        atb(new LoseHPAction(adp(), adp(), magicNumber, AbstractGameAction.AttackEffect.FIRE));
```

Comment:
I wouldn't say Infernal Blaze has a stronger effect than Immolate.  Infernal Blaze damage is weaker the turn you play it, equivalent the next turn, and passes Immolate after that.  Many fights that I want Immolate I'm clearing low health enemies so I'd rather the damage up front than some extra damage later.  Also, Consume is more of a downside than adding a burn to the discard pile.

I'll think about Flaming Leap.  A big downside of Consume isn't just that you transform a card into garbage, it is that you have to have a card to Consume which means not playing a card in your hand.  With Flaming Leap's draw that's less of a downside.  So I may make this a different effect altogether.

I have done a lot of runs and Burn Bright sounds better on paper than it actually is.  It will be unpickable at 2 cost.  It is potentially quite strong but it has conditions to reach that level of strength.  It is hard to consistently double a specific power early in the fight because you might draw Burn Bright before what you want to double.  You have to have a consumable card in your hand that you don't play that turn.  It exhausts so it isn't spammable.  The last two are ways that it is weaker than Amplify.

Reply:
The thing about the comparison of Infernal Blaze and Immolate is that being slightly slower to deal damage is going to be more than compensated for by dealing a lot more damage in the fights that go longer (which will include most elite and boss fights). I think that's a positive tradeoff for many runs, perhaps most runs. And there's a lot of times where you'll have innate powers (including the one from this pack) or a high enough power density to consistently hit good stuff with Burn Bright. I do get where you're coming from, but it's different from how I would evaluate these cards.

Whether a deck has enough draw to be okay with the Consume downside on these cards will vary, but drawing more cards is something you want to be doing anyway, and a lot of decks end up with a few cards that aren't playing each turn. It's also that all the numbers in this pack are pushed to be very high, so you're going to have access to other very strong cards to complement each other. You won't be able to completely stuff your deck with them due to the Consume downsides, but you can still fit in a decent amount.

For what it's worth, I did another playtesting run with this pack (which I don't do for most packs I review, I did it here since I felt the pack was interesting). This time I took it all the way to beating the Heart. I took most of the Fueled pack cards I saw early, including 2 Burning Strikes that were great all the way through the endgame and a Conflagration that was also good throughout (it was particularly helpful as scaling damage for the act 1 boss).

I also picked up Flaming Leap close to the end of the run and it was as strong as I expected. I did see some of the other Fueled cards off 2 Destiny Draw, and picking Infernal Blaze from that felt better than picking Immolate (only had it as an option 2 or 3 times over the course of the run, but it stood out in strength). I also got Fuel Tank pretty late in the run (middle of act 3) and it was quite good, really increased the flexibility of the Consume cards.

The pack is definitely fun to play with so it's in a good state from that perspective. The only card I saw and didn't really think I'd pick is the new dice rolling card. Unless I think I need something like that to win or have strong healing/temporary HP, I'm pretty reluctant to take a card that is likely to make me lose HP for the entire fight. I would have been very happy to get a Controlled Burn but never saw one.

If you don't want to change the numbers of Infernal Blaze or Burn Bright I'm not going to insist on it, since it's a new pack and there's room for reasonable differences in how we evaluate its power level. I think we're reasonably likely to get feedback on its power level once we release it, but releasing this pack and seeing whether we actually get that feedback or not is fine. If we do, we'll adjust the numbers down, so be aware of that possibility.